docs/source/usage/workflows/boundaryConditions.rst

```diff
@@ -15,15 +15,20 @@ Particles
 
 By default, boundary kinds match the value of ``--periodic``.
 For species with a particle pusher, it can be overridden with option `<prefix>_boundary <x> <y> <z>`.
```

Comment:
Could you also fix the formatting of this parameter and add -- so that it matches the `--periodic` above? And same for boundaryOffset below.